src/components/ReportCard/ReportCard.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Collapse from '@material-ui/core/Collapse';
+import './ReportCard.scss';
+class ReportCard extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = { expanded: false };
+    this.handleExpandClick=this.handleExpandClick.bind(this);
+  }
+  handleExpandClick(){
+    this.setState({ expanded: !this.state.expanded });
+  }
+  render() {
+    const { data, meetingNumber } = this.props;
+    const title = "מפגש ".concat(meetingNumber.toString());
+    return (
+      <Card className="report-card">
+        <CardHeader title={title} onClick={this.handleExpandClick}>
+          <IconButton
+            onClick={this.handleExpandClick}
+            aria-expanded={this.state.expanded}
+            aria-label="Show more"
+          >
+            <ExpandMoreIcon />
+          </IconButton>
+        </CardHeader>
+        <Collapse in={this.state.expanded} timeout="auto" unmountOnExit>
+          <CardContent className="report-card-content">
+            <TextField id="location" label="מיקום המפגש" value={data ? data.location : null} variant="outlined" />
+            <TextField id="date" label=" תאריך המפגש" value={data ? data.date : null} type="date" variant="outlined" />
+            <TextField id="description" variant="outlined" label="תיאור המפגש" defaultValue='' value={data ? data.description : null} multiline />
+          </CardContent>
+          <CardActions>
+            <Button size="small" color="primary">ביטול</Button>
+            <Button size="small" color="primary">שמור</Button>
+          </CardActions>
+        </Collapse>
+      </Card>
+    );
+  }
+}
+
+ReportCard.propTypes = {
+  data: PropTypes.object,
+  meetingNumber: PropTypes.number,
+};
+ReportCard.defaultProps = {
+  data: null,
```

Comment:
i would consider going with 
```suggestion
  data: {
    location: '',
    date: '',
    description: ''
 }    
```
 if u decide to take my comment above with the shape proptype then this is a must actually 😅